### PR TITLE
Adding new shortcodes

### PR DESCRIPTION
**Pull request by matt-auckland**
# [ 🔜  Preview Here]()

# Changes 

- Swaps site over to the `nav-community-v2` theme (tested to check nothing breaks)
- Adds a new page: www.navcoin.org/shortcodes : This is an example of the short codes that can be used on the website
- Adds a new shortcode `center_text_cta`
- Adds new file which documents how to use the shortcodes `navcoin-org/themes/nav-community-v2/docs/how-to-use-shortcodes.md`
- New button type: 'gradient-btn`
- Various CSS tweaks to improve the site's appearance

## Fixes

- Empty `<p></p>` tags that are direct children of the body tag are now hidden (fixes a lot of whitespace issues)
- Deletes a lot of unused JS and CSS

### CODE REVIEW

File: themes/nav-community-v2/layouts/partials/head.html [hunk 1]
```diff
@@ -49,13 +40,13 @@
     <meta name="twitter:card" content="summary">
     <meta name="twitter:title" content="NavCoin">
     <meta name="twitter:description" content="NavCoin is a decentralized cryptocurrency that uses peer-to-peer technology to operate with no central authority or banks; managing transactions and the issuing of NavCoin is carried out collectively by the network. NavCoin is open-source; its design is public, nobody owns or controls NavCoin and everyone can take part.">
-    <meta name="twitter:image" content="https://navcoin.org/images/logo-extended.webp">
+    <meta name="twitter:image" content="https://navcoin.org/images/nav_blue-trend.png">
```
Comment (matt-auckland) [inline themes/nav-community-v2/layouts/partials/head.html]: Not all web browsers can load webp so I've swapped the image

File: themes/nav-community-v2/layouts/partials/head.html [hunk 2]
```diff
@@ -49,13 +40,13 @@
...
 
     <!-- Open Graph data -->
     <meta property="og:site_name" content="NavCoin">
     <meta property="og:title" content="NavCoin">
     <meta property="og:description" content="NavCoin is a decentralized cryptocurrency that uses peer-to-peer technology to operate with no central authority or banks; managing transactions and the issuing of NavCoin is carried out collectively by the network. NavCoin is open-source; its design is public, nobody owns or controls NavCoin and everyone can take part.">
-    <meta property="og:image" content="https://navcoin.org/images/logo-extended.webp">
+    <meta property="og:image" content="https://navcoin.org/images/nav_blue-trend.png">
```
Comment (matt-auckland) [inline themes/nav-community-v2/layouts/partials/head.html]: Not all web browsers can load webp so I've swapped the image

File: themes/nav-community-v2/theme.toml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-homepage = "http://example.com/"
+homepage = "http://navcoin.org"
 tags = []
 features = []
-min_version = "0.37"
+min_version = "0.1"
 
 [author]
-  name = "Paul Sanderson"
+  name = "NavCoin Core Team"
```
Comment (matt-auckland) [inline themes/nav-community-v2/theme.toml]: ❤️ 😍 ❤️

File: themes/nav-community-v2/layouts/partials/head.html [hunk 4]
```diff
@@ -18,29 +18,20 @@
     <link rel="shortcut icon" href="/images/favicon-1.png">
     <!-- Site Title -->
 
-    <!-- google font -->
-    <link href="https://fonts.googleapis.com/css?family=Patua+One|Raleway|Roboto|Source+Sans+Pro" rel="stylesheet">
+    <!-- google fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Raleway|Roboto|Source+Sans+Pro" rel="stylesheet">
```
Comment (matt-auckland) [inline themes/nav-community-v2/layouts/partials/head.html]: We weren't using this font at all so I removed it

**Comment by red010b37** [comment]
Re-deployed

File: themes/nav-community-v2/theme.toml (hunk 3 above)
Comment (red010b37) [reply themes/nav-community-v2/theme.toml]: :sob:      :hamburger:

**Pull request merged**